src/backend/env/user.rs

```diff
@@ -554,11 +554,10 @@ impl User {
     }
 
     pub fn eligible_for_minting(&self) -> bool {
-        self.mode == Mode::Mining && !self.controversial() &&
-            // While `controversial` covers users with negative rewards, user with 0 rewards should
-            // not be picked for minting because they either have no rewards, or they were
-            // downvoted back to zero and then nothing should be minted for them.
-            self.rewards() > 0
+        self.mode == Mode::Mining
+            && !self.controversial()
+            && self.rewards() > 0
+            && self.active_weeks > 0
```

Comment:
Yep, I assumed no one would be bothered by this and my main worry would be that if e.g. a user has died and would never log in again, then minting tokens to their account would be kind of stupid.

But if you think this could be sensitive, I'll run a poll in /DAO, thanks!

Reply:
https://6qfxa-ryaaa-aaaai-qbhsq-cai.icp0.io/#/post/1115929